src/olympia/bandwagon/models.py

```diff
@@ -450,9 +450,7 @@ def post_save_or_delete(sender, instance, **kw):
 class CollectionUser(models.Model):
     collection = models.ForeignKey(Collection)
     user = models.ForeignKey(UserProfile)
-    role = models.SmallIntegerField(
-        default=1,
-        choices=amo.COLLECTION_AUTHOR_CHOICES.items())
+    role = models.SmallIntegerField(default=1)  # TODO: drop me.
```

Comment:
We'll be dropping a bunch of Collection models properties after frontend takes over so we can drop this one at the same time.

Reply:
Let's add a when, why and reference an github issue too please

Reply:
https://github.com/mozilla/addons-server/pull/6494/commits/3a06d116dbe80282f6ff2673678a0ad699f1baef